prymer/offtarget/bwa.py

```diff
@@ -188,7 +188,7 @@ class BwaResult:
 MAX_GAP_OPENS: int = 0
 """The default maximum number of gap opens allowed in the full query sequence"""
 
-MAX_GAP_EXTENSIONS: int | None = None
+MAX_GAP_EXTENSIONS: int = 6
```

Comment:
The changes in this file seem totally unrelated - could you back them out please?  I'd say put them in a separate PR, but I'm heavily editing this file on a separate branch, and we can handle it there.

Reply:
This is needed since the updated `pybwa` dependency has breaking changes to the constructor API: https://github.com/fulcrumgenomics/pybwa/releases/tag/2.0.0